conan.lock

```diff
@@ -5,7 +5,7 @@
         "websocketpp/0.8.2#842a0419153a8aa52f3ea3a1da557d38%1695972005.713",
         "tinygltf/2.5.0#65c28d0a4c3cbd4ef92b08b59df769da%1701621757.442",
         "termcap/1.3.1#1986f84bf21dd07ea774b027a3201fcb%1678542508.75",
-        "swig/4.1.1#8f418438101ef7712d52a9fe739d32f9%1707838298.077707",
+        "swig/4.1.1#2bb5c79321cbb05bcab525c690d9bf74%1716336914.081294",
```

Comment:
Tweaked and built from https://github.com/jmarrec/conan-center-index/commit/52e12ba5e491462dfb4bba5e86c4c141db2735e9